main.py

```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
```

Comment:
The story is supposed to loop 10 times, so we should initialize loop to 0 and change the while condition to <= 9.